### PR TITLE
Remove unnecessary entries in .cfg files

### PR DESCRIPTION
**Pull request by no-lex**
Applies to 25 maps, saving ~180 KB per map, or about 4.5 MB total
.mpz files included because texturecull requires applicable .mpz files as well (so the map and config have matching texture mappings)

### CODE REVIEW

**Review by TheAssassin — CHANGES_REQUESTED** [review]
How are those shader variable values unnecessary?
Apart from that: You also changed some of the other variables, too (e.g. [this one](https://github.com/red-eclipse/maps/pull/31/commits/ce84ae5e76f1fd8878e6684fb0d1d48046b7efd3#diff-b4fb4ae2406bae500b863d86456b9d89R11)). That's not very atomic, I wouldn't merge this PR.

**Comment by no-lex** [comment]
I didn't manually change them, I think that's just the result of re-saving an old map in a newer version.
It's as atomic as i can get with the current version of the game, I think.

The culled lines are references to the full texture list, when only about 50-100 textures are included in the map. The unnecessary ones were removed with /texturecull.

**Comment by TheAssassin** [comment]
Well, in that case we don't have to trust your words that this is correct but the code @qreeves wrote. Looks OK to me then.

**Review by TheAssassin — APPROVED** [review]

**Comment by no-lex** [comment]
By the way, this seems to remove the invalid `setpixelparam` setting from the luckystrike-barlight texture that was causing the invalid commands on map load. Since this gets ignored with the current version, this change won't change (or fix) the lack of that shader.

I don't exactly know what `setpixelparam 0 0.800000 0.800000 0.800000 0.000000` does, so I'm not sure how to fix it (revert it to when `setpixelparam` worked)

File: absorption.cfg [hunk 1]
```diff
@@ -144,14 +150,17 @@ mapmusic "sounds/music/track_06"
...
 // waterspec 150
 // watersubdiv 3
 // yawcloudlayer 0
 // yawclouds 0
 // yawenvlayer 0
 // yawsky 0
 
+gname = [editing]
+modetexs = [<grey>textures/modes/editing ]
+mut = [131072]
```
Comment (qreeves) [inline absorption.cfg]: These lines are from the loading screen interface and shouldn't be in the map file. I thought I fixed this problem but it still seems to happen.

**Review by qreeves — CHANGES_REQUESTED** [review]
Need to remove UI variables from map configs.

**Comment by no-lex** [comment]
Forgot about that, should be fixed now.

**Pull request merged**